src/main/java/uk/ac/cam/cl/dtg/isaac/dos/users/RegisteredUser.java

```diff
@@ -79,15 +80,18 @@ public class RegisteredUser extends AbstractSegueUser {
      *            - whether the user has verified their email or not
      */
     @JsonCreator
-    public RegisteredUser(@JsonProperty("id") final Long id,
+    public RegisteredUser(
+            @JsonProperty("id") final Long id,
             @JsonProperty("givenName") final String givenName, @JsonProperty("familyName") final String familyName,
             @JsonProperty("email") final String email, @JsonProperty("role") final Role role,
             @JsonProperty("dateOfBirth") final Date dateOfBirth, @JsonProperty("gender") final Gender gender,
             @JsonProperty("registrationDate") final Date registrationDate,
             @JsonProperty("lastUpdated") final Date lastUpdated,
             @JsonProperty("emailToVerify") final String emailToVerify,
             @JsonProperty("emailVerificationToken") final String emailVerificationToken,
-            @JsonProperty("emailVerificationStatus") final EmailVerificationStatus emailVerificationStatus) {
+            @JsonProperty("emailVerificationStatus") final EmailVerificationStatus emailVerificationStatus
+//            @JsonProperty("teacherPending") final Boolean teacherPending
```

Comment:
Does this need to be commented out? It's likely to cause a merge conflict I think